packages/firebase_remote_config/ios/firebase_remote_config.podspec

```diff
@@ -1,6 +1,11 @@
 #
 # To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
 #
+
+require 'yaml'
+pubspec = YAML.load_file(File.join('..', 'pubspec.yaml'))
+ffVersion = pubspec['version'].gsub('+', '-')
```

Comment:
I would name this something other than ffVersion as this feels like a non standard variable name. Perhaps just libraryVersion

Reply:
used `libraryVersion`